docs/tech/casepro.md

```diff
@@ -0,0 +1,12 @@
+# CasePro
+
+[CasePro](https://github.com/rapidpro/casepro) is an open-source
+project from the same organisation that builds RapidPro.
+
+## Pods
+
+CasePro pods allow you to display relevant information in the CasePro
+interface without merging it into the CasePro codebase.
```

Comment:
Currently we can only add information in the case detail view (so you have to open a case for a message before you can see those details). Maybe we should make a note of this?